src/features/tags/components/TagsAside.js

```diff
@@ -25,27 +27,25 @@ class TagsAside extends Component {
 
     let visibleTags = activeTags === "new" ? mostRecentTags : topTags;
     let isHidden = activeTags === "" ? "hidden" : "";
-    let activeTop = activeTags === "top" ? "primary" : "secondary";
-    let activeNew = activeTags === "new" ? "primary" : "secondary";
+    let activeTop = activeTags === "top" ? "contained" : "text";
+    let activeNew = activeTags === "new" ? "contained" : "text";
 
     return (
       <>
-        <div className="button-group button-group--block">
-          <button
+          <Button 
+            variant={activeTop}
             onClick={this.handleClick}
-            className={`button button--${activeTop}`}
             name="top"
           >
             Top Tags
-          </button>
-          <button
+          </Button>
+          <Button
+            variant={activeNew}
             onClick={this.handleClick}
-            className={`button button--${activeNew}`}
             name="new"
           >
             New Tags
-          </button>
-        </div>
+          </Button>
         <TagsList tags={visibleTags} isHidden={isHidden} />
```

Comment:
I don't see the TagsList component in this update. Could you update it to use MUI's [List](https://mui.com/components/lists/#main-content) component instead of a general div?

Reply:
I like the look with tabs, but I don't know if it's necessary to wrap the tabs with a list. I was referring to the [TagsList](https://github.com/zero-to-mastery/breads-client/blob/master/src/features/tags/components/TagsList.tsx) component with my previous comment about updating to use a MUI list